packages/app/src/styles/theme/_apply-colors-light.scss

```diff
@@ -241,6 +241,13 @@ $dropdown-link-active-bg: $bgcolor-dropdown-link-active;
 }
 
 .grw-user-page-list-m {
+  .expand-compress-btn {
+    color: $body-color;
+    background-color: $body-bg;
+  }
```

Comment:
In case that this style is the same as both light and dark modes, this should be written on apply-colors.scss.